src/index.js

```diff
@@ -1,37 +1,149 @@
 import "./style.css";
+
+class toDoTask {
+  constructor(desc, index, completed = false) {
+    this.desc = desc;
+    this.index = index;
+    this.completed = completed;
+  }
+}
+
 class UI {
   static displayTask() {
-    const task = [
-      {
-        desc: "buy fruits",
-        index: 1,
-        completed: false,
-      },
-      {
-        desc: "buy medicines",
-        index: 2,
-        completed: false,
-      },
-      {
-        desc: "buy household",
-        index: 3,
-        completed: false,
-      },
-    ];
-
-    task.forEach((works) => UI.addTasks(works));
+    // const task = [
+    //   {
+    //     desc: "buy fruits",
+    //     index: 1,
+    //     completed: false,
+    //   },
+    //
+    // ];
+    const Alltasks = Store.getTasksFromStore();
+
+    Alltasks.forEach((individualTask) => UI.addTasksToList(individualTask));
   }
 
-  static addTasks(works) {
+  static addTasksToList(individualTask) {
     const list = document.querySelector(".list");
-    let html = ` <li class="listItems">
+    let html = ` <li class="listItems" id="${individualTask.index}">
     <input type="checkbox" name="select" id="" class="check" />
-    <p class="title">${works.desc}</p>
-    <i id="moreicon" class="fa-solid fa-ellipsis-vertical"></i>
+    <input type="text" class="taskInput taskDesc" value="${individualTask.desc}" />
+    <i id="moreicon" class="fa-solid fa-trash-can"></i>
+    
   </li>`;
     list.insertAdjacentHTML("beforeend", html);
   }
+
+  static removeTask(element) {
+    if (element.classList.contains("fa-trash-can")) {
+      element.parentElement.remove();
+    }
+  }
+}
+
```

Comment:
- [ ] From the project requirements, the following new functionalities should be implemented in a separate file and then later imported in the `index.js` file This will help you showcase your skills in  JavaScript ES6 modular feature and import/export feature 😎 👍

Reply:
done